cypress/integration/stoplight-project.ts

```diff
@@ -3,32 +3,122 @@ describe('Stoplight component', () => {
     cy.redirectUnpkgToLocalDistFolder('elements');
   });
 
-  it('loads correctly', () => {
-    loadStoplightProjectPage();
+  describe('Api page', () => {
+    it('loads api page correctly', () => {
+      loadStoplightProjectPage();
+      cy.findByText('To-dos').click();
+      cy.findByText('https://todos.stoplight.io/').should('exist');
+      cy.findByText('https://stoplight.io/mocks/elements-examples/studio-demo/389434').should('exist');
+      cy.findByRole('heading', { name: /API Key/ }).should('exist');
+      cy.findByText('💫 Overview').should('exist');
+    });
   });
 
-  it('navigates via table of contents', () => {
-    loadStoplightProjectPage();
-    cy.findByText('pets').click();
-    cy.findByText('Create a pet').click();
-    cy.findByRole('heading', { name: /Create a pet/i, timeout: 30000 }).should('exist');
+  describe('Operation page', () => {
+    it('loads operation docs correctly', () => {
+      loadCreateTodoPage();
+      cy.findByRole('heading', { name: /Create Todo/i }).should('exist');
+      cy.findByRole('heading', { name: /Request/i }).should('exist');
+      cy.findByRole('heading', { name: /post \/todos/i }).should('exist');
+      cy.findByRole('heading', { name: /Response/i }).should('exist');
+    });
+
+    it('does not break on select dropdown', () => {
+      loadCreateTodoPage();
+      cy.findByRole('button', { name: /request sample/i }).click();
+      cy.findByText('Obj-C').should('exist');
+    });
+
+    it('displays request sample correctly', () => {
+      loadCreateTodoPage();
+      cy.findByLabelText(
+        'curl --request POST \\ --url \'https://todos.stoplight.io/todos?apikey=\' \\ --header \'Content-Type: application/json\' \\ --data \'{ "name": "string", "completed": false }\'',
+      ).should('exist');
+    });
+
+    it('displays example correctly', () => {
+      loadCreateTodoPage();
+      cy.findByLabelText(
+        '{ "id": 0, "name": "string", "completed": true, "completed_at": "2019-08-24T14:15:22Z", "created_at": "2019-08-24T14:15:22Z", "updated_at": "2019-08-24T14:15:22Z" }',
+      ).should('exist');
+    });
+
+    it('displays schema correctly', () => {
+      loadCreateTodoPage();
+      cy.findByText('id').should('exist');
+      cy.findByText('completed_at').should('exist');
+    });
+
+    it('invokes TryIt request', () => {
+      loadCreateTodoPage();
+      cy.findByRole('button', { name: /send request/i }).click();
+      cy.findByText('403 Forbidden').should('exist');
+      cy.findByText('"NOT AUTHORIZED"').should('exist');
+    });
+
+    it('mocks response correctly', () => {
+      loadListTodosPage();
+      cy.findByRole('button', { name: /mocking/i }).click();
+      cy.findByRole('menuitemcheckbox', { name: /enabled/i }).should('exist');
+      cy.findByRole('menuitemcheckbox', { name: /enabled/i }).then(enabled => {
+        enabled.click();
```

Comment:
This is a kind of hack to make it work as the  `menuitemcheckbox` element is detached from dom when invoking regular `click` by chained cypress api.